src/networking.c

```diff
@@ -2700,6 +2700,18 @@ NULL
                 return;
             }
 
+            if (options & CLIENT_TRACKING_BCAST) {
+                int collision_idx = findPrefixCollisions(c,prefix,numprefix);
+                if (collision_idx != -1) {
+                    addReplyErrorFormat(c,
+                    "Prefix '%s' overlaps with one of the existing prefixes "
```

Comment:
maybe we can return which prefix this has conflict with and showing that to user? if we do so maybe we can pass the pointer into findPrefixCollisions and set the other conflicting prefix in the function

Reply:
i don't mind returning details about the conflict, but i don't think it is a must (i.e. if it adds a lot of code, we can live without it).

Reply:
I think it'll only be a couple of lines, so I'll do it.